lab3/forker.c

```diff
@@ -4,7 +4,7 @@
 
 int main(void)
 {
-    for (int i = 0; i < 5; i++)
+    for (int i = 0; i < 15; i++)
```

Comment:
I had set this to 5 because my raspberry pi was maxing out all LEDs very quickly, so with a lower number it takes less time before the load starts to drop.  Did 15 work better on your raspberry pi?